game-core/src/main/java/games/strategy/engine/lobby/moderator/toolbox/tabs/moderators/ModeratorsTabModel.java

```diff
@@ -26,29 +25,29 @@
 
   private final ToolboxModeratorManagementClient toolboxModeratorManagementClient;
 
-  @Getter private final boolean isSuperMod;
+  @Getter private final boolean isAdmin;
 
   ModeratorsTabModel(final ToolboxModeratorManagementClient toolboxModeratorManagementClient) {
     this.toolboxModeratorManagementClient = toolboxModeratorManagementClient;
-    isSuperMod = toolboxModeratorManagementClient.isCurrentUserSuperMod();
+    isAdmin = toolboxModeratorManagementClient.isCurrentUserAdmin();
   }
 
   List<String> fetchTableHeaders() {
-    return isSuperMod() ? SUPER_MOD_HEADERS : HEADERS;
+    return isAdmin() ? SUPER_MOD_HEADERS : HEADERS;
```

Comment:
SUPER_MOD_HEADERS should get renamed as well

Reply:
I was surprised how many places said 'super-mod'

Updated: 7b83c1af6